src/main/java/com/github/instagram4j/instagram4j/requests/status/StatusSetStatusRequest.java

```diff
@@ -0,0 +1,69 @@
+package com.github.instagram4j.instagram4j.requests.status;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.instagram4j.instagram4j.IGClient;
+import com.github.instagram4j.instagram4j.models.IGPayload;
+import com.github.instagram4j.instagram4j.requests.IGPostRequest;
+import com.github.instagram4j.instagram4j.responses.IGResponse;
+import com.github.instagram4j.instagram4j.responses.accounts.AccountsUserResponse;
+
+import lombok.Data;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.Accessors;
+
+@RequiredArgsConstructor
+public class StatusSetStatusRequest extends IGPostRequest<IGResponse> {
```

Comment:
Appreciate the changes! One last change needed is a constructor for StatusSetStatusRequest for default fields.

A constructor for the payload is fine, but I would a constructor for the fields directly from the request constructor

In short I should be able to do
```new StatusSetStatusRequest(String text, String emoji, long expires_at)```
and
```new StatusSetStatusRequest(String text, String emoji, long expires_at, boolean should_notify, String status_type)```

I believe this should be trivial to add, so after these two small changesthe PR will be merged!

Again thanks for contributing

Reply:
No problem, I changed it!